examples/gw150914/audio.py

```diff
@@ -0,0 +1,30 @@
+from pycbc.frame import read_frame
+from pycbc.filter import highpass_fir, lowpass_fir
+from pycbc.psd import welch, interpolate
+from pycbc.types import TimeSeries
+import urllib
+import pylab
+
+# Read data and remove low frequency content
+fname = 'H-H1_LOSC_4_V2-1126259446-32.gwf'
+url = "https://losc.ligo.org/s/events/GW150914/" + fname
+urllib.urlretrieve(url, filename=fname)
```

Comment:
Is this the "right" (TM) way of getting LOSC data? Shouldn't we ask the LOSC server where it is in some way and then download from there rather than using a hardcoded filepath?